toonz/sources/translations/chinese/image.ts

```diff
@@ -101,7 +101,7 @@ Please try raising the FFmpeg timeout in Preferences.</source>
     </message>
     <message>
         <location filename="../../image/sgi/filesgi.cpp" line="1235"/>
-        <source>Endianess</source>
+        <source>Endianness</source>
```

Comment:
Thank you @alexmyczko for the typo fixes.
.ts files (the _translation source_ files) are automatically generated from the source.
Unless the typos are fixed in the original source, they will be overwritten by wrong words at some points in the future.

Can you please find the corresponding typos in the source and fix them as well?
For instance, this case of the word  `Endianess` , the original source is located in [the line 1235 of toonz/sources/image/sgi/filesgi.cpp](https://github.com/opentoonz/opentoonz/blob/master/toonz/sources/image/sgi/filesgi.cpp#L1235) as you can see in the `location` tag above this line.

Reply:
I tried, but I'm not sure if I edited too much or not, can you check? I wasn't able to find the other typos anywhere else though.